apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -367,6 +372,10 @@
                     "obsolete" : {
                         "description" : "Old endpoints that will be removed",
                         "type" : "boolean"
+                    },
+                    "dry-run" : {
```

Comment:
so:
```yaml
http:
  endpoints:
    dry-run: true
```
 should enable the new endpoint without disabling the other endpoints ?
I'm asking because I never used this configuration path and the schema does not say what's it's type? Is it object or list ? What's the default.

Reply:
Shouldn't we also set the defaults here ? I see there is no defaults of the old endpoints, but it's never late to fix it :)

Reply:
yes, this is how you enable it.